docs/components/dartpad/menus/dropdown/main.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Dropdown Menu Demo',
+      debugShowCheckedModeBanner: false,
+      home: DropdownMenuDemo(),
+    );
+  }
+}
+
+class DropdownMenuDemo extends StatefulWidget {
+  @override
+  _DropdownMenuDemoState createState() => _DropdownMenuDemoState();
+}
+
+class _DropdownMenuDemoState extends State<DropdownMenuDemo> {
+  Map<int, bool> countToValue = <int, bool>{};
```

Comment:
This looks like it isn't being used, I think you can remove it and make this a statelesswidget instead

Reply:
Done